File: application/src/main/java/com/example/server/service/CommentService.java
package com.example.server.service;

import com.example.api.dto.CommentDto;
import com.example.server.repository.entity.Comment;

/**
 * Сервис для взаимодействия с отзывами.
 */
public interface CommentService {

    /**
     * Добавление нового отзыва.
     * @param itemId идентификатор предмета
     * @param commentDto информация об отзыве
     * @param userId идентификатор пользователя
     */
    Comment create(Long itemId, CommentDto commentDto, Long userId);
}

File: application/src/main/kotlin/com/example/server/service/CommentService.kt
package com.example.server.service

import com.example.api.model.CommentDto
import com.example.server.entity.Comment

/**
 * Сервис для взаимодействия с отзывами.
 */
interface CommentService {

    /**
     * Добавление нового отзыва.
     *
     * @param itemId идентификатор предмета
     * @param commentDto информация об отзыве
     * @param userId идентификатор пользователя
     */
    fun create(itemId: Long, commentDto: CommentDto, userId: Long): Comment
}
